<?xml version="1.0"?>
<launch>
    <arg name="name" doc="The environment name."/>
    <arg name="node_name" default="node" doc="Set the node name."/>
    <group ns="$(arg name)">
        <node name="$(arg node_name)" pkg="eagerx_core" type="rxnode" output="screen"/>
    </group>
</launch>